design-documents/graph-ql/coverage/ProductPrices.graphqls

```diff
@@ -0,0 +1,52 @@
+interface ProductInterface{
+    ...
+    price: ProductPrices @deprecated (use price_range)
+    price_range: PriceRange!
+    tier_price: Float @deprecated (use tier_prices)
+    tier_prices: [ProductTierPrices]
+    ...
+}
+
+type ProductPrices { @deprecated (use PriceRange)
+    minimalPrice: @deprecated
+    maximalPrice: @deprecated
+    regularPrice: @deprecated
+}
+
+type ProductTierPrices {
+    customer_group_id: String
+    qty: Float
+    value: Float @deprecated (use final_price)
+    final_price: Money
+    percentage_value: Float @deprecated (use discount)
+    discount: ProductDiscount
+    website_id: Float
+}
+
+type PriceRange{
+    minimum_price : ProductPrice!
+    maximum_price : ProductPrice
+}
+
+type ProductPrice{
+    regular_price : Price!
+    final_price : Price!
+    discount : ProductDiscount
+}
+
+type ProductDiscount{
+    percent_off : Float
+    amount_off : Float
+}
+
+#No changes, just for reference
+
+type Price @doc(description: "The Price object defines the price of a product as well as any tax-related adjustments.") {
+    amount: Money @doc(description: "The price of a product plus a three-letter currency code.")
+    adjustments: [PriceAdjustment] @doc(description: "An array that provides information about tax, weee, or weee_tax adjustments.")
```

Comment:
Last time we discussed prices with Nishant Kapoor we agreed to remove adjustments from the storefront.

Reply:
+ @nrkapoor

Reply:
@akaplya We will need adjustments for wee and wee_tax

Reply:
ok, so we can not separate regular taxes as well?

Reply:
We can but don't see a need to do that for catalog.

Reply:
so, let rename adjustments to wee? this will be more transparent

Reply:
@nrkapoor In case we do not separate taxes from an amount why do we need to separate weee tax form weee? This looks like inconvenient.
if we are removing taxes from the storefront let's include weee tax into weee.
your opinion?

Reply:
also, if we need a WEEE only, we could move this field on ProductPrice level. WEEE is the same for regular and final prices.

Reply:
that makes sense. can you share how the schema would look like with the change you are proposing?

Reply:
Made changes regarding Price/Adjustments/Taxes